tests/api_tests.py

```diff
@@ -43,6 +43,14 @@ def dummy_wait(self, cond, timeout=None):
 Motor.wait = dummy_wait
 
 class TestAPI(unittest.TestCase):
+
+    def setUp(self):
+        # micropython does not have _testMethodName
+        try:
+            print("\n\n{}\n{}".format(self._testMethodName, "=" * len(self._testMethodName,)))
+        except AttributeError:
+            pass
```

Comment:
This isn't needed to suppress the warnings, I just added it so I could tell what tests were generating the warnings.  I kinda like the output but I can chop this if others do not.